third_party/wabt/src/interp/jit/jit.cc

```diff
@@ -44,6 +48,62 @@ void JITFunction::call() const {
 
   func.func_entry = func_entry_;
   func.code(context, reinterpret_cast<void*>(start));
+
+  StackAllocator* stackAllocator = new StackAllocator(0);
+  for (ValueType result_type : result_types) {
+    stackAllocator->push(LocationInfo::typeToValueInfo(result_type));
+  }
+
+  int data_index = 0;
+  std::vector<LocationInfo>& offsets = stackAllocator->values();
+#define push_result(type)                                  \
+  results.push_back(Value::Make(*(reinterpret_cast<type*>( \
+      reinterpret_cast<u8*>(data) + offsets[data_index].value))));
+
+  for (ValueType result_type : result_types) {
+    switch (result_type) {
+      case Type::I32:
+        push_result(s32);
+        break;
+      case Type::I64:
+        push_result(s64);
+        break;
+      case Type::F32:
+        push_result(f32);
+        break;
+      case Type::F64:
+        push_result(f64);
+        break;
+      case Type::V128:
+        push_result(v128);
+        break;
+      case Type::I8:
+        push_result(s8);
+        break;
+      case Type::I16:
+        push_result(s16);
+        break;
+      case Type::FuncRef:
+      case Type::ExternRef:
+      case Type::Reference:
+      case Type::Func:
+        push_result(Ref);
+        break;
+      case Type::I8U:
+        push_result(u8);
+        break;
+      case Type::I16U:
+        push_result(u16);
+        break;
+      case Type::I32U:
```

Comment:
I don't think these types can be result values. They are memory types as far as I know.